tests/src/python/test_qgsserver_wms_getmap_size_server.py

```diff
@@ -43,7 +43,7 @@ def setUp(self):
         os.environ['QGIS_SERVER_WMS_MAX_HEIGHT'] = '3000'
         super(TestQgsServerWMSGetMapSizeServer, self).setUp()
         self.project = os.path.join(self.testdata_path, "test_project_with_size.qgs")
-        self.expected_too_big = self.strip_version_xmlns(b'<ServiceExceptionReport version="1.3.0" xmlns="http://www.opengis.net/ogc">\n <ServiceException code="InvalidParameterValue">The requested map size is too large</ServiceException>\n</ServiceExceptionReport>\n')
+        self.expected_too_big = self.strip_version_xmlns(b'<ServiceExceptionReport version="1.3.0" xmlns="http://www.opengis.net/ogc">\n <ServiceException code="Size error">The requested map size is too large</ServiceException>\n</ServiceExceptionReport>\n')
```

Comment:
same as above, but I guess this mean that the source of the exception should be fixed too.

Reply:
I tought so when I first saw it, but #9646 is not back ported to 3.6 if I see correctly?

Reply:
i.e. fix  `throw QgsBadRequestException( QStringLiteral( "Size error" )` in line 803 of src/server/services/wms/qgswmsrenderer.cpp

Reply:
yeah, no backports for #9646 , I think we must fix these minor issues one by one as they occour.

Reply:
so should I fix the qgswmsrenderer.cpp and revert the change in the code?

Reply:
Yes, please, "Size error" is not an allowed value for WMS (SLD) specs, the correct one is `InvalidParameterValue`, I know it's not related to this PR but IMO is better to fix it than leave a bug and  wrong value in the test.